### ISSUE TITLE
LLM_Type='moss'： The model weights are not tied. Please use the `tie_weights` method before using the `infer_auto_device` function

### ISSUE THREAD

**Issue by lhtpluto** [wenda-LLM/wenda#407]
启动参数如下：
(D:\temp ai\wenda\env) D:\temp ai\wenda>python wenda.py -t moss
Namespace(Config='config.yml', LLM_Type='moss', Logging=None, Port=None)

启动报错如下：

The model weights are not tied. Please use the `tie_weights` method before using the `infer_auto_device` function.

**Comment by lhtpluto** [comment]
知识库加载完成
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 4/4 [00:12<00:00,  3.18s/it]
The model weights are not tied. Please use the `tie_weights` method before using the `infer_auto_device` function.
The model weights are not tied. Please use the `tie_weights` method before using the `infer_auto_device` function.
模型加载完成

**Comment by lhtpluto** [comment]
提问后，回答问题异常：
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:106068 for open-end generation.
Both `max_new_tokens` (=2050) and `max_length`(=2050) seem to have been set. `max_new_tokens` will take precedence. Please refer to the documentation for more information. (https://huggingface.co/docs/transformers/main/en/main_classes/text_generation)

**Comment by liuyake** [comment]
用了baichuan-7b也是同样的问题，楼主解决了吗？ @lhtpluto

**Comment by LCG22** [comment]
> 用了baichuan-7b也是同样的问题，楼主解决了吗？ @lhtpluto

我也是 baichuan-7b 遇到同样的问题

**Comment by yixin-zhu** [comment]
我用moss遇到同样的问题
